app/javascript/components/assets/mockApartments.js

```diff
@@ -25,5 +25,18 @@ let apartments = [
     pets: "No pets",
     user_id: 1,
   },
+  {
```

Comment:
This looks great!  I like the fact that you are testing data for different users!